mm2src/coins/rpc_command/get_new_address.rs

```diff
@@ -472,7 +472,7 @@ pub(crate) mod common_impl {
 
         Ok(GetNewAddressResponse {
             new_address: HDAddressBalance {
-                address: address.to_string(),
+                address: coin.address_formatter()(&address),
```

Comment:
Is it possible to add a test coverage for this format (or for the entire response, or both)?

Reply:
There is test coverage for the format and it was the reason I added the `address_formatter` trait method in the past when the below failed https://github.com/KomodoPlatform/komodo-defi-framework/blob/df3d39c1ab4017ff8498a4028bc66b551b0c9c3f/mm2src/mm2_main/tests/docker_tests/docker_tests_inner.rs#L5292-L5295 But I will add test coverage for this RPC for EVM (There is coverage for UTXO only for this method).

Reply:
Added to an already existing test to avoid duplication https://github.com/KomodoPlatform/komodo-defi-framework/pull/2264/commits/18cab0d1b3f4d05fc83d6bdc245c1cf3826613cd